lib/generators/rest_api_generator/resource_generator.rb

```diff
@@ -15,20 +15,20 @@ class ResourceGenerator < Rails::Generators::NamedBase
 
     def create_service_file
       Rails::Generators.invoke("model", [file_name, build_model_attributes])
-      if options["scope"].empty?
-        controller_path = "#{API_CONTROLLER_DIR_PATH}/#{file_name.pluralize}_controller.rb"
+      # if options["scope"].empty?
+        controller_path = "#{define_scope}/#{file_name.pluralize}_controller.rb"
         controller_test_path = "#{API_TEST_DIR_PATH}/#{file_name.pluralize}_controller_spec.rb"
         template "rest_api_controller.rb", controller_path
         template "rest_api_spec.rb", controller_test_path
         routes_string = "resources :#{file_name.pluralize}"
         route routes_string
-      else
-        scope_path = options["scope"].pluralize
-        controller_path = "#{API_CONTROLLER_DIR_PATH}/#{scope_path}/#{file_name.pluralize}_controller.rb"
-        controller_test_path = "#{API_TEST_DIR_PATH}/#{scope_path}/#{file_name.pluralize}_controller_spec.rb"
-        template "child_api_controller.rb", controller_path
-        template "child_api_spec.rb", controller_test_path
-      end
+      # else
+      #   scope_path = options["scope"].pluralize
+      #   controller_path = "#{API_CONTROLLER_DIR_PATH}/#{scope_path}/#{file_name.pluralize}_controller.rb"
+      #   controller_test_path = "#{API_TEST_DIR_PATH}/#{scope_path}/#{file_name.pluralize}_controller_spec.rb"
+      #   template "child_api_controller.rb", controller_path
+      #   template "child_api_spec.rb", controller_test_path
+      # end
```

Comment:
Children options was removed for now, alright? Can you just delete this code for now? In next features we will add this back